### DISCUSSION TITLE
handbook/integration/jenkins-ci/

### DISCUSSION THREAD

**Discussion by giscus** [serenity-js/serenity-js#2124] [Comments]
# handbook/integration/jenkins-ci/

Serenity/JS integrates with industry-standard test runners, so that your test scenarios can be executed

https://serenity-js.org/handbook/integration/jenkins-ci/

**Comment by BorisTB** [comment]
yo how do I get a thousand thumbs from ten thousand lightning bugs because i really want this piece of documentation
